src/transformers/models/t5/modeling_t5.py

```diff
@@ -491,7 +493,7 @@ def project(hidden_states, proj_layer, key_value_states, past_key_value):
             # if key and values are already calculated
             # we want only the last query position bias
             if past_key_value is not None:
-                position_bias = position_bias[:, :, -seq_length:, :]
```

Comment:
Couldn't we just do `-int(seq_length)` here?

Reply:
It should but I'm not sure, some stuff if surprising in `torch.export`.

I think it's ok to just merge this.